shoppingcart/migrations/0005_new_initial.py

```diff
@@ -1,45 +1,97 @@
 # Generated by Django 4.2.20 on 2025-09-19 05:43
```

Comment:
I think we can add migrations to the "excluded" things in Black. Being a machine generated file, there is no benefit to formatting it with a formatter. Also, regnerating them for minor changes will create very noisy diffs.

Reply:
> That file is actually partially manually created

Ah! Then it makes sense to prefer formatting. 

> I'd don't see how it will make noisy diffs?

Now that I think about it, it's mostly solved at the PR review time. I run into noisy diff situations when working on a new feature. I would add an attribute and create a migration, then adjust something and recreate the same migration, and my editor or git diff would show everything has changed in the file, instead of single line that I would like to see. Think of fixing the spelling of something for example.

Anyway, I guess I have built it into "machine generated means don't change" thumb rule.